### PR TITLE
Add stuck queue check to health status

### PR DESCRIPTION
**Pull request by Genza999**
This PR adds a task stuck queue check for checking pending tasks that:
- Have lasted a maximum threshold as pending
- Have exceeded the maximum number of pending tasks allowed

The implementation shoould cause the health check page to error if there are N amount of unprocessed tasks or if there are unprocessed tasks older than X time.
The task is : https://app.clickup.com/t/14289126/FC-2768

### CODE REVIEW

File: src/fullctl/django/settings/__init__.py [hunk 1]
```diff
@@ -197,6 +197,9 @@ def set_default_v1(self):
 
         self.set_option("TASK_RECHECK_DECAY_MAX", 3600)
 
+        self.set_option("MAX_PENDING_TASKS", 100)
+        self.set_option("TASK_MAX_AGE_THRESHOLD", 24)
```
Comment (vegu) [inline src/fullctl/django/settings/__init__.py:201]: @Genza999 lets add a comment noting unit size (hours)

File: src/fullctl/django/settings/__init__.py [hunk 2]
```diff
@@ -197,6 +197,9 @@ def set_default_v1(self):
 
         self.set_option("TASK_RECHECK_DECAY_MAX", 3600)
```
Comment (vegu) [inline src/fullctl/django/settings/__init__.py:198]: @Genza999 while you're in here please also add a comment here stating this is seconds and controls the maximum re-evaluation decay for task qualification

File: src/fullctl/django/views/__init__.py [hunk 3]
```diff
@@ -36,6 +38,25 @@ def diag(request):
     return HttpResponse(mark_safe(f"<div><pre>Meta:\n{escape(txt)}</pre></div>"))
 
 
+def check_task_stack_queue():
+    """
+    Tests the task stack queue
+    """
+    pending_tasks = Task.objects.filter(status="pending")
```
Comment (vegu) [inline src/fullctl/django/views/__init__.py:45]: @Genza999 lets add `queue_id__isnull=True` to the filter here, so we only check tasks that haven't been picked up by a task worker yet. (e.g., the queue is getting backed up)

**Review by vegu — CHANGES_REQUESTED** [review]

File: src/fullctl/django/views/__init__.py (hunk 3 above)
Comment (Genza999) [reply src/fullctl/django/views/__init__.py]: Added, thanks!

File: src/fullctl/django/settings/__init__.py (hunk 1 above)
Comment (Genza999) [reply src/fullctl/django/settings/__init__.py:205]: Added, thanks!

**Review by vegu — APPROVED** [review]

**Comment by vegu** [comment]
PR looks good, but i think we need to make sure all task queues are cleaned before we deploy this, so retargeting to `draft-pr-branch` for now.

**Comment by vegu** [comment]
Lets rebase and adjust this for the changes in https://github.com/fullctl/fullctl/pull/226 once that's merged

**Comment by vegu** [comment]
something about the commit history is janky, will reopen a new pr

**Pull request merged**